travis/run_pylint.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import os
+import sys
+
+import click
+import pylint.lint
+
+CLICK_DIR = click.Path(exists=True, dir_okay=True, resolve_path=True)
+
+
+def get_count_fails(linter_stats):
+    """Verify the dictionary statistics to get number of errors.
+    :param linter_stats: Dict of type pylint.lint.Run().linter.stats
+    :return: Integer with quantity of fails found.
+    """
+    count = 0
+    for msg in linter_stats['by_msg']:
```

Comment:
Yes, here return all checks activated in cfg and found.
If a check is returned then is a fail. (Currently bash script is working equal)
If you dont want a fail then you remove the check from cfg file